Cinebase/Use Cases/Movies/MovieDetail/Showtimes/ShowtimesCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ShowtimesCell.swift
+//  Cinebase
+//
+//  Created by Charles Moncada on 5/16/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import UIKit
+
+protocol ShowtimesCellDelegate: class {
+    func cellTapped(sender: ShowtimesCell, schedule: String)
+}
+
+class ShowtimesCell: UITableViewCell {
+    
+    @IBOutlet weak var cinemaLabel: UILabel!
+   
+    @IBOutlet weak var scheduleSelectionControl: TagsSelectionControl!
+    
+    let schedule = ["21:40","22:40"]
+    
+    var delegate: ShowtimesCellDelegate?
+    
+    var cineData: String? {
+        didSet {
+            cinemaLabel.text = cineData
+        }
+    }
+    
+    override func awakeFromNib() {
+       setupScheduleSelectionControl(with: schedule)
+    }
+    
+    func setupScheduleSelectionControl(with schedule: [String]) {
+        scheduleSelectionControl.spacing = 5
+        
+        scheduleSelectionControl.labels = schedule
+        
+        scheduleSelectionControl.tagHeight = 25
+        scheduleSelectionControl.paddingX = 10
+        
+        scheduleSelectionControl.cornerRadius = 5
+        scheduleSelectionControl.tagBackgroundColor = .orange
+        scheduleSelectionControl.tagSelectedBackgroundColor = .red
+    }
+    
+    @IBAction func scheduleSelectionTapped(_ sender: TagsSelectionControl) {
+        let schedule = sender.tagViews[sender.selectedViewIndex].tagNameLabel.text!
```

Comment:
I think the delegate should receive the selectedViewIndex instead the text. That way it should be easier to map it to a model.